WooCommerce/Classes/POS/Presentation/ItemCardView.swift

```diff
@@ -5,12 +5,28 @@ struct ItemCardView: View {
     private let item: POSItem
     private let onItemCardTapped: (() -> Void)?
 
+    @ScaledMetric private var scale: CGFloat = 1.0
+
     init(item: POSItem, onItemCardTapped: (() -> Void)? = nil) {
         self.item = item
         self.onItemCardTapped = onItemCardTapped
     }
 
     var body: some View {
+        HStack {
+            if let imageSource = item.productImageSource {
+                ProductImageThumbnail(productImageURL: URL(string: imageSource),
+                                      productImageSize: 60,
+                                      scale: scale,
+                                      productImageCornerRadius: 1,
+                                      foregroundColor: .clear)
+            } else {
+                // TODO:
+                // Handle what we'll show when there's lack of images:
```

Comment:
Maybe we can get some nice Woo placeholder graphic

Reply:
We could, but I'd prefer to not import or reference assets in the `app` side at this point, specially if we're going to remove them with the i3 design 🙇